src/site/pages/digital/src/containers/ExprToCircuitPopup/index.scss

```diff
@@ -0,0 +1,115 @@
+@import "~shared/utils/theme";
+
+.exprToCircuitPopup {
+    padding: 0px;
+    margin: 0px;
+    top: 0px;
+    left: 0px;
+
+    /* Using 576px since that is the smallest cutoff for Bootstrap */
+    @media (min-width: 576px) {
+        margin: 0px 50px;
+    }
+
+    label {
+        vertical-align: middle;
+        color: $secondary-color;
+    }
+
+    button {
+        line-height: 30px;
+        height: 30px;
+        margin-top: 5px;
+        padding-right: 7px;
+        padding-left: 7px;
+        cursor: pointer;
+        transition: 0.5s;
+        color: #fff;
+        border-radius: 4px;
+        float: right;
+
+        &:enabled:hover {
+            color: #ccc;
+            border-color: #aaa;
+        }
+    }
+
+    .generate {
+        background-color: $success-color;
+
+        &:enabled:hover {
+            background-color: darken($success-color, 10%);
+        }
+    }
+
+    .cancel {
+        background-color: $danger-color;
+        margin-right: 10px;
+
+        &:enabled:hover {
+            background-color: darken($danger-color, 20%);
+        }
+    }
```

Comment:
All subclasses need to be like

`&__subclassname`

the `&` inherits the parent's name, i.e. the parent in this case is `.exprToCircuitPopup`, so the child would be come `.exprToCircuitPopup__generate` or `.exprToCircuitPopup__cancel`.

We do this because otherwise, it infects the global classname space, especially with common names like "cancel". 
If some other file had a classname called "cancel" for some cancel button, it would inherit the same styles as this one which could be very confusing.

Could be worth having a global `.cancel` class though, in something like
https://github.com/OpenCircuits/OpenCircuits/blob/master/src/site/pages/digital/src/containers/App/index.css
or maybe
https://github.com/OpenCircuits/OpenCircuits/blob/master/src/site/shared/components/Popup/index.scss

Reply:
Moved .cancel to https://github.com/OpenCircuits/OpenCircuits/blob/master/src/site/shared/components/Popup/index.scss